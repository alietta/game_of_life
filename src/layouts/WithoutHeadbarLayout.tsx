import React from 'react';
import { Container, useTheme } from 'sancho';
import { Layout } from './LayoutStyles';

export const WithoutHeadbarLayout: React.FC = (props) => {
  const theme = useTheme();
  return (
    <Layout background={theme.colors.background.default}>
      <Container css={{ maxWidth: 1024, height: '100%' }}>
        {props.children}
      </Container>
    </Layout>
  );
};
