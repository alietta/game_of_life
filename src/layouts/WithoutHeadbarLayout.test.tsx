import React from 'react';
import { mount } from 'enzyme';
import { WithoutHeadbarLayout } from './WithoutHeadbarLayout';

describe('WithoutHeadbarLayout', () => {
  const Component: React.FC = () => <h1>Test</h1>;
  it('render', () => {
    const element = mount(
      <WithoutHeadbarLayout>
        <Component />
      </WithoutHeadbarLayout>
    );
    expect(element.html()).toMatchSnapshot();
  });
});
