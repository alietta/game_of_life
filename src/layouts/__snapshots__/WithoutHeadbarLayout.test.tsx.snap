// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithoutHeadbarLayout render 1`] = `"<div class=\\"css-1jel6y2-Layout e1hm601p0\\"><div class=\\"css-13ru0hl-WithoutHeadbarLayout\\"><h1>Test</h1></div></div>"`;
